python/src/numerous/frameworks/fastapi.py

```diff
@@ -0,0 +1,30 @@
+"""Module for integrating Numerous with FastAPI."""
+
+from fastapi import Request
+
+from numerous import user_session
+from numerous.local import is_local_mode, local_user
+
+
+class FastapiCookieGetter:
```

Comment:
Can we name this `FastAPICookieSetter`?

Reply:
You mean FastAPICookieGetter right? I don't think its setting any cookies, but the Capitalization of API I agree to :-)